utils/Constants.ts

```diff
@@ -130,13 +132,24 @@ export const rskMultisigTreasuryAddress = "0x18bc35c3b74b35c70cff0ec14ad62f4a8c2
 export const bdxLockAmount = to_d18(3150000);
 export const bdxLockingContractAddressRSK = "0x4292Ef0D3AfA1052605e2D706349dFe3A481cDcF";
 
-export const ETH_USDC_ADDRESS = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"; // usdc on eth
+type SupportedToken = {
+  [key: string]: { symbol: string; address: string; decimals: number };
+};
 
-export const EXTERNAL_USD_STABLE: { [key: string]: { symbol: string; address: string; decimals: number } } = {
+export const ETH_USDC_ADDRESS = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"; // USDC on eth
+export const EXTERNAL_USD_STABLE: SupportedToken = {
   mainnetFork: { symbol: "USDC", address: ETH_USDC_ADDRESS, decimals: 6 },
   rsk: { symbol: "XUSD", address: RSK_XUSD_ADDRESS, decimals: 18 }
 };
 
+export const ETH_DAI_ADDRESS = "0x6B175474E89094C44Da98b954EedeAC495271d0F"; // DAI on eth
+export const SECONDARY_EXTERNAL_USD_STABLE: SupportedToken = {
+  mainnetFork: { symbol: "DAI", address: ETH_DAI_ADDRESS, decimals: 18 },
+  rsk: { symbol: "DOC", address: RSK_DOC_ADDRESS, decimals: 18 }
+};
+
+export const EXTERNAL_SUPPORTED_TOKENS = [EXTERNAL_USD_STABLE, SECONDARY_EXTERNAL_USD_STABLE];
```

Comment:
Let's change the name to indicate these are specifically external USD stables and not general external tokens